app/questionnaire/navigation.py

```diff
@@ -50,8 +50,20 @@ def build_navigation(self, current_group_id, current_group_instance):
                 else:
                     logger.debug("building navigation", group_id=group['id'])
                     navigation.append(self._build_single_navigation(group, current_group_id, first_location))
+
+        if self._all_sections_completed(navigation):
```

Comment:
@benjefferies If I understand correctly, this could be achieved by moving all confirmation/summary blocks into their own group at the end of every schema that we currently have and removing the `hide_in_navigation` property from these blocks.

Then some additional logic would be required in `build_navigation` to check that all blocks (including any hidden groups) on the routing path are in `completed_blocks`?